packages/sandbox/src/helpers/subscriptionPostMessageRelayHelpers.ts

```diff
@@ -1,45 +1,60 @@
+import EventEmitter from 'eventemitter3';
 import type { ExecutionResult } from 'graphql';
 import { Client, createClient as createGraphQLWSClient } from 'graphql-ws';
 import {
   Observer,
   OperationOptions,
   SubscriptionClient as TransportSubscriptionClient,
 } from 'subscriptions-transport-ws';
-import type { JSONObject, JSONValue } from './types';
 import {
   EXPLORER_SET_SOCKET_ERROR,
   EXPLORER_SET_SOCKET_STATUS,
   EXPLORER_SUBSCRIPTION_RESPONSE,
   EXPLORER_SUBSCRIPTION_TERMINATION,
 } from './constants';
 import {
+  executeOperation,
+  HandleRequest,
   sendPostMessageToEmbed,
   SocketStatus,
 } from './postMessageRelayHelpers';
-import type { ObjMap } from 'graphql/jsutils/ObjMap';
+import type { JSONObject } from './types';
 
 export type GraphQLSubscriptionLibrary =
   | 'subscriptions-transport-ws'
-  | 'graphql-ws';
+  | 'graphql-ws'
+  | 'http-multipart';
 
 // @see https://www.typescriptlang.org/docs/handbook/advanced-types.html#exhaustiveness-checking
 function assertUnreachable(x: never): never {
   throw new Error(`Didn't expect to get here ${x}`);
 }
 
-class SubscriptionClient {
-  protocol: GraphQLSubscriptionLibrary;
+type HTTPMultipartParams = {
+  includeCookies?: boolean;
+  handleRequest: HandleRequest;
+};
+
+export type HTTPMultipartClient = EventEmitter<
+  'connected' | 'error' | 'disconnected'
+> & {
+  stopListeningCallback: (() => void) | undefined;
+};
+
```

Comment:
ahh missed that bundle bump, it looks like eventemitter3@3.1.2 might already be included by one of the transport protocols, can you try using `3.1.2` instead of `5.0.0` and see if that removes that 73kb?